codalab/worker/local_run/local_dependency_manager.py

```diff
@@ -87,45 +89,77 @@ def _save_state(self):
             self._state_committer.commit({'dependencies': self._dependencies, 'paths': self._paths})
 
     def _load_state(self):
+        """
+        Load states from dependencies-state.json file that contains bundles with other information (e.g., stage,
+        dependency, last_used, etc.) and populates values for self._dependencies, self._dependency_locks, and self._paths
+        """
         state = self._state_committer.load(default={'dependencies': {}, 'paths': set()})
+
         dependencies = {}
```

Comment:
These variables don't seem to be used, remove?

Reply:
They are used in the next few lines.